checkov/terraform/checks/resource/aws/ECRImageScanning.py

```diff
@@ -5,7 +5,7 @@
 class ECRImageScanning(BaseResourceValueCheck):
     def __init__(self):
         name = "Ensure ECR image scanning on push is enabled"
-        id = "CKV_AWS_161"
+        id = "CKV_AWS_163"
```

Comment:
@njgibbon Is `CKV_AWS_161` used for any other check? I'm not sure it is

Reply:
Hey @nimrodkor  - There actually isn't if you check the generated docs in the main branch. But, turns out we are in a slightly awkward state somehow as the docs haven't updated correctly...

I believe changes went in yesterday that all individually passed their checks. But, caused a conflict as they went in. 

https://github.com/bridgecrewio/checkov/pull/1241/files

By putting this change through. This should all be resolved with no conflict on 161, 162 or 163.